keps/sig-node/2040-cri-beta/README.md

```diff
@@ -0,0 +1,264 @@
+    - [Beta -&gt; GA Graduation](#beta---ga-graduation)
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+  - [Version Skew Strategy](#version-skew-strategy)
+- [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+  - [Feature Enablement and Rollback](#feature-enablement-and-rollback)
+  - [Rollout, Upgrade and Rollback Planning](#rollout-upgrade-and-rollback-planning)
+  - [Monitoring Requirements](#monitoring-requirements)
+  - [Dependencies](#dependencies)
+  - [Scalability](#scalability)
+  - [Troubleshooting](#troubleshooting)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Release Signoff Checklist
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [x] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [x] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+Promote the CRI API to beta as it has been in use in production for a while.
+
+## Motivation
+CRI based runtimes such as CRI-O and containerd have been in use in production for over a year now with the current CRI API.
+We want to signal to the users that the CRI API is production ready and they should feel comfortable moving away dockershim
+as it is slated to be deprecated.
+
+### Goals
+- Graduate the CRI API to beta.
+- Identify any fields that need to made more type safe such as Seccomp.
+- Address and cleanup the notes/todos in the CRI.
+
+### Non-Goals
+- Block on any big new features.
+
+## Proposal
+Promote the CRI API version to v1beta and have more than one CRI runtime implement the changes made as part of the move to beta.
+
+### Risks and Mitigations
+There may be performance regressions in using CRI stats vs. cadvisor for gather container stats. The beta period will be used to dive
+into the details and make sure that the performance is at par before moving on to GA. Meanwhile, kubelet has a way to still use cadvisor
+for stats as a mitigation.
+
+## Design Details
+
+### Identified Work Items
```

Comment:
Add a work item to no longer map the following flags as experimental:
container-runtime-endpoint

I think we should keep the image-service-endpoint experimental and evaluate if it makes sense or not to just remove it at the moment as a configurable flag.  I am curious if anyone is using the flags with separate values.

Reply:
Added.